pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mkdocs-multirepo-plugin"
-version = "0.6.3"
+version = "0.6.4"
```

Comment:
Can you remove this change? Wanting to combine a few other PRs into one new release. CI automatically generates a release based on this value.

Reply:
removed as requested.  Thanks for the plugin!